tree/ntuple/v7/test/ntuple_storage.cxx

```diff
@@ -434,6 +434,7 @@ TEST(RPageSinkBuf, Basics)
    FileRaii fileGuard("test_ntuple_sinkbuf_basics.root");
    {
       RNTupleWriteOptions options;
+      options.SetEnablePageChecksums(false); // disable same page merging
```

Comment:
Should same page merging be fully coupled to page checksums, or do you think there might be cases where page checksums are desired without page checksums? If so, I would consider adding a separate toggle for same page merging (which in turn also enables page checksums). If not, perhaps this function could be renamed to `SetSamePageMerging` to better communicate the effect of enabling it.

Reply:
That's a good point. I thought about it and I came to the conclusion that I don't see a good reason why users would, essentially, want to turn off getting smaller files as it can be done at negligible cost. This, and the fact that when we one option conditionally depends on another, the option setters and getters become quite a bit more complicated (warning messages for invalid option combinations etc.).

Reply:
Okay yes, that makes sense. I'm still not really sure if in this case it should be called `SetEnablePageChecksums` or `SetSamePageMerging`, because from the first name it is not clear that this will give the additional optimization (i.e. someone looking to optimize their storage might not take into account this setting). We can probably help a hand by clearly documenting this side effect in the `RNTupleWriteOptions` (and potentially `tuning.md`?).